corehq/apps/app_manager/suite_xml/case_tile_templates/README.md

```diff
@@ -0,0 +1,14 @@
+# Case Tile Templates
+
+Each template consists of a `name.xml` template file and a `name.json` metadata
+file in this directory, and a corresponding attribute on the `CaseTileTemplates`
+choices class.
+
+For new templates, I propose following these conventions:
```

Comment:
what a great idea, agreed with all of these

Reply:
ooh, that's nice

Reply:
I would add that we should use underscores to connect vertical and horizontal since we have precedence for that. I also wonder if the `map` field needs a qualifier stating that it will not show up in the grid itself or not?

Reply:
good points - I mostly wanted to get _something_ down, figuring we can iterate on this as we build out more templates.  Feel free to add whatever additions you feel are appropriate!